src/Collection.php

```diff
@@ -245,14 +258,13 @@ public function aggregate(array $pipeline, array $options = [])
      */
     public function bulkWrite(array $operations, array $options = [])
     {
-        if (! isset($options['writeConcern']) && ! is_in_transaction($options)) {
-            $options['writeConcern'] = $this->writeConcern;
-        }
+        $options = $this->inheritWriteOptions(
+            $this->inheritCodec($options),
+        );
```

Comment:
I realize the signatures allow these to be chained, but I think it'd be more readable on separate lines. Would make it easier to grep for things like `inheritWriteOptions($options)` and would also minimize future diffs if we modify inheritance logic.

Reply:
I removed chained invocations in favour of re-assigning `$options`.